cmd/compose/compose.go

```diff
@@ -311,7 +312,9 @@ func RootCommand(dockerCli command.Cli, backend api.Service) *cobra.Command { //
 		"commandConn.CloseRead:",
 	))
 
-	opts := ProjectOptions{}
+	opts := ProjectOptions{
+		Dependencies: types.IncludeDependencies,
```

Comment:
I wonder `IgnoreDependencies` should be the default value, and we opt-in for dependencies in up/start/stop/down commands